src/pykeen/losses.py

```diff
@@ -1462,20 +1462,31 @@ def forward(
         num_entities: Optional[int] = None,
     ) -> torch.FloatTensor:  # noqa: D102
         # neg scores might be -inf for masked values -> get rid of those
-        mask = torch.isfinite(neg_scores)
-        neg_weights, neg_scores = neg_weights[mask], neg_scores[mask]
+        # mask = torch.isfinite(neg_scores)
+        # neg_weights, neg_scores = neg_weights[mask], neg_scores[mask]
```

Comment:
had to comment those two lines to be able to do sum over columns (negative scores) of a batch